benchmarks/6.1.0/v1/6.1.0_single-node_2-core_jdk-11.md

```diff
@@ -1,3 +1,17 @@
+# V1
+In this version of Performance results, the tests were executed with the following configs added which disable some of the newly added features/improvements. This has been done in order to ease cross-comparison with earlier versions of the product. Refer [this](https://github.com/wso2/performance-is/tree/master/benchmarks/6.1.0/v1/README.md) for more details.
```

Comment:
Lets use relative paths in all the places.. It will smooth the navigation with forks and less error prone to content structure.

Reply:
Fixed with 8c4c46faf17b695c5157bfde5c54b2533d7d0492